exodus-installer.sh

```diff
@@ -46,9 +51,9 @@ exodus_is_installed() {
 
 exodus_uninstall() {
   # remove app files
-  rm -f /usr/bin/Exodus
+  rm -f  /usr/bin/Exodus
```

Comment:
Extra space?

Reply:
Yeah, sorry, I like to make the arguments line up:
```
  rm -f  /usr/bin/Exodus
  rm -rf /opt/exodus
  rm -f  /usr/share/applications/Exodus.desktop
```

Reply:
I can take them out if you like?